renderer/components/upscayl-tab/view/ProgressBar.tsx

```diff
@@ -27,13 +31,14 @@ function ProgressBar({
     <div className="absolute z-50 flex h-full w-full flex-col items-center justify-center bg-base-300/50 backdrop-blur-lg">
       <div className="flex flex-col items-center rounded-btn bg-base-100/50 p-4 backdrop-blur-lg">
         <Logo className="spinner mb-4 h-12 w-12" />
-        <p className="rounded-full px-2 pb-2 font-bold">
+        <p className="rounded-full px-2 pb-2 font-bold flex flex-col items-center">
           {batchMode && "Batch Upscale In Progress: " + batchProgress}
 
           {!batchMode &&
             (doubleUpscaylCounter > 0
               ? `${progress}\nPass ${doubleUpscaylCounter}`
               : `${progress}`)}
+          <NPBC value={progressFloat} />
```

Comment:
It would also be more productive to just compute the progress number on the fly here like
```suggestion
          <NPBC value={parseFloat(progress.replace("%", ''))} />
```
This is much more efficient, plus we don't need regex-replaceall here since it should be just one %. Potentially in the future a slight optimization could be made here to just take the first 4 or 5 chars.

This would also entail removing progressFloat everywhere.
You can add both suggestions here to batch to commit both of them.